packages/web-components/src/components/va-memorable-date/va-memorable-date.scss

```diff
@@ -25,6 +25,10 @@
   margin-bottom: 0;
 }
 
+.usa-hint {
+  display: block;
+}
```

Comment:
The [USWDS markup for memorable date](https://designsystem.digital.gov/components/memorable-date/) uses a `span` which naturally displays inline. But our variation has added a `legend` element which the span is now added to. This means that it was displaying in line with the label text.

I added this overriding style to fix that alignment issue but we could also just change the `span` to a `div`. If we feel like that's a better approach instead of style overrides, I can remove this and update the markup.